radicle-cli/src/commands/checkout.rs

```diff
@@ -13,7 +13,7 @@ use crate::terminal::args::{Args, Error, Help};
 
 pub const HELP: Help = Help {
     name: "checkout",
-    description: "Checkout a radicle project working copy",
+    description: "Checkout a project's working copy",
```

Comment:
What about 'checkout a project into the local directory"?  The term "working copy" might not be clear for newcomers to the project.